package.json

```diff
@@ -9,8 +9,16 @@
     "build": "grunt build",
     "docs": "grunt yui",
     "test": "grunt",
+    "fix": "grunt eslint:fix",
```

Comment:
To fix what?
I would prefer to be more descriptive.

`lint:fix` will be good.
And no `eslint:fix` please, because we have changed linting tool recently and we can do this in the future.
With `lint` script name we can do this whenever we want and the command will remain unchanged.

Reply:
Absolutely!